pkg/charts/linkerd2/values_test.go

```diff
@@ -240,3 +241,37 @@ func TestNewValues(t *testing.T) {
 		}
 	})
 }
+
+// TestHAValuesParsing tests whether values commonly used in HA deployments have
+// appropriate types and can be successfully parsed.
+func TestHAValuesParsing(t *testing.T) {
+	// come up with some values that cover most of the fields we have in there.
```

Comment:
Do we need this comments inside the block?

Reply:
Nope, good catch.